samples/javascript/E2_CopyFileToBlob/index.js

```diff
@@ -2,7 +2,7 @@ const fs = require("fs");
 const path = require("path");
 const storage = require("azure-storage");
 
-module.exports = function (context, filePath) {
+module.exports = async function (context, filePath) {
     const container = "backups";
     const root = path.parse(filePath).root;
     const blobPath = filePath
```

Comment:
Out of scope, but probably would suggest making const blobService in the scope of this file and not of the function, want to persist the blob service for as long as possible (and not create on each function invocation)

Reply:
Not out of scope; good reminder!